static/usage/v7/input/migration/angular.md

```diff
@@ -0,0 +1,11 @@
+```html
```

Comment:
This and the other `migration` files aside from `index.md` don't appear to be used. I'm guessing these were created and then never deleted when it turned out you didn't need them?